File: tests/fixtures/cpp/else_if_chain.cpp
// Expected metrics:
// classify: cognitive=TBD, cyclomatic=TBD, sloc=TBD
const char* classify(int x) {
    if (x > 100) {
        return "high";
    } else if (x > 50) {
        return "medium";
    } else if (x > 0) {
        return "low";
    } else {
        return "negative";
    }
}

File: tests/fixtures/go/else_if_chain.go
package fixtures

// Expected metrics:
// classify: cognitive=TBD, cyclomatic=TBD, sloc=TBD
func classify(x int) string {
	if x > 100 {
		return "high"
	} else if x > 50 {
		return "medium"
	} else if x > 0 {
		return "low"
	} else {
		return "negative"
	}
}
